src/plugins/ccode/ccode.hpp

```diff
@@ -11,27 +11,35 @@
 
 #include <kdbplugin.h>
 
+#ifdef __cplusplus
```

Comment:
Is there any reason to include this in C code?

Reply:
Since C does not support `using` or `extern "C"` we have to remove the code below, if we include the header file in `testmod_ccode.c`.